lib/index.js

```diff
@@ -0,0 +1,28 @@
+const ChromeSimpleTextWrapper = require('./chromeSimpleTextWrapper');
+const ChromeRobustTextWrapper = require('./chromeRobustTextWrapper');
+
+// supported browsers
+const CHROME_BROWSER = 'chrome';
```

Comment:
talvez a gente pudesse abstrair o `browser` e considerar o comportamento. Por exemplo, eu não lembro bem mas, `STRIP_LAST_SPACE`. O ponto é que talvez mais browser tenha o mesmo comportamento

Reply:
Concordo com você. O problema é que não temos conhecimento de quais comportamentos diferem em cada browser. A gente teria que dedicar um tempo e fazer essa análise. Como não é o foco agora, achei mais conveniente e funcional a gente simplesmente assumir que a implementação do algoritmo é específica para cada browser. Conforme a gente for expandindo o leque de browsers, dá pra gente generalizar o algoritmo e parametrizar os comportamentos.
Você acha que devemos fazer isso agora ou mais pra frente?

Reply:
realmente, gastar muito tempo não seria o foco agora. Isso me fez pensar se o mesmo comportamento é compartilhado em um browser em diferentes SOs. Aquilo que a gente notou só foi testado no Chrome no Mac. Seria bom pelo menos fazer um teste super rápido em Windows.

Reply:
Testei aqui no Windows e a saída do Chrome é a mesma. 🙏